Sources/Components/Gallery/GalleryComponent.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct GalleryComponent: Component {
+    public let id: String
+    public let placeholder: UIImage
+    public let stringURLs: [String]
+
+    public init(id: String = UUID().uuidString, placeholder: UIImage, stringURLs: [String]) {
```

Comment:
Could the placeholder image possibly have a default vale as the asset you added? Or do we not want it to hold images..?

Reply:
I think it's better to not have a default placeholder.